src/components/keyboard/NumberButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Dimensions, Pressable, StyleSheet, Text } from 'react-native';
+import { useTheme } from '../../styles/themes';
+import type { KeyboardActions } from './Keyboard';
+
+const { height: SCREEN_HEIGHT } = Dimensions.get('screen');
+
+interface Props {
+    value: string;
+    onPress: (action: KeyboardActions, input?: string) => void;
+}
+
+export const NumberButton = ({ value, onPress }: Props) => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        button: {
+            height: SCREEN_HEIGHT * 0.0671,
+            width: '33.3%',
```

Comment:
Same here: this value is copied on NumberButton, DeleteButton and SubmitButton.

I may be wrong, but I think it would be more clear to have the size of the buttons managed by the Keyboard component.

In fact, I think that if we just represent each button rows with a dedicated `View` with `display:flex;flex-direction:row`, and fill theses rows with 3 buttons they will automatically take a 1/3 of the parent size.

Ex:

```html
<Keyboard>

```

Reply:
But same goes if you apply to parent component wrap and flexdirection:row, and set the width of child elements. 
so for example if i wanted to have 4 elements in one row, i would set it to 25% and etc.

Reply:
Yeah, this solution handles every possible number of buttons per row depending on the size you set for the row's elements.
But in our case there will always be 3 button per row and we don't plan to add other buttons with different width.

Reply:
Alright, you are right there. I will change it